### DISCUSSION TITLE
Site fails to load when user was deleted

### DISCUSSION THREAD

**Discussion by tionis** [requarks/wiki#3223] [Help / Questions] [answered]
**Describe the bug**
If a user is deleted while still being logged in, they cannot access any page.

**To Reproduce**
1. Create a user
2. Log in as the user in another session
3. Delete the user
4. Try to access any page

**Expected behavior**
The user is logged out and can only access pages available to guests.

**Host Info (please complete the following information):**
 - OS: Docker
 - Wiki.js version: 2.5.191
 - Database engine: Postgres 11.11

**Comment by tionis** [comment]
Why was this issue closed?

**Reply by NGPixel** [reply]
It wasn't closed, it was converted to a discussion. As this isn't a bug.

**Comment by NGPixel** [comment, marked as answer]
This is the expected behavior. JWTs are valid for the duration of the short expiration. Either shorten the duration under Admin > Security or regenerate the keys under Utilities to log everybody out.

**Reply by tionis** [reply]
Ok, thanks for the explanation!
